app/client/rpc/handlers.go

```diff
@@ -0,0 +1,56 @@
+package rpc
```

Comment:
KISS and wanting to show where I was going fast by using mostly a codebase (V0) that you guys already are familiar with but that I don't know much about.

It's easy to switch library while I learn more about the requirements. 
For example, you mentioned that it would be nice to have the ability to generate the RPC server code starting from an openAPI specification. 

I am going to simplify the codebase quite a bit and also implement this nice feature (along with the Makefile command to generate/regenerate the server code whenever we'll update the spec).

The next update of this PR tomorrow will have this too together with the fixes suggested by you guys, for which I thank you both obviously.

Sneak peek: 
![image](https://user-images.githubusercontent.com/29378614/188507631-0cd75f4e-0aef-4dcd-9b5b-b02b4a23122b.png)